vm/devices/serial/serial_debugcon/src/lib.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Implementation of the bochs / QEMU debugcon device.
+//!
+//! This is a zero-configuration, output-only serial device, which should only
+//! be used for debugging (hence the name). It offers no flow control
+//! mechanisms, or any method of reading data into the Guest.
+
+#![warn(missing_docs)]
```

Comment:
Side note: maybe it's time to make this the default and add allows instead.

Reply:
I bet @smalis-msft would have a good idea on what the current composition of documented vs. undocumented we currently have.

But in any case - I do agree that having a stricter documentation policy moving forwards (now that we are open-source) might be a good idea.

Reply:
Might be a good idea to track this via an Issue as well

Reply:
#287